myia/compile/backends/__init__.py

```diff
@@ -52,11 +52,11 @@ def pytorch_default(device="cpu:0"):
 
 _backends = {
     "relay": (
-        channel_loader("myia.compile.backends.relay", "RelayBackendR"),
+        channel_loader("myia.compile.backends.relay", "RelayBackend"),
```

Comment:
You might want to rename channel_loader to something else.

Reply:
Renamed to backend_loader.